src/TwilioVideo.ios.js

```diff
@@ -176,6 +182,12 @@ export default class TwilioVideo extends Component {
     this._stopLocalAudio();
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.localVideoTrackName !== this.props.localVideoTrackName) {
```

Comment:
🎉 Good call on updating here.

Reply:
So crazy to see a `componentDidUpdate` instead of `useEffect` now 😆

Reply:
😎